code/game/objects/items/storage/boxes.dm

```diff
@@ -1316,6 +1316,15 @@
 	for(var/i in 1 to 4)
 		new/obj/item/reagent_containers/food/snacks/chewable/bubblegum/happiness(src)
 
+/obj/item/storage/box/gum/bubblegum
+	name = "bubblegum gum packet"
+	desc = "The packaging is entirely in demonic, apparently. You feel like even opening this would be a sin."

```

Comment:
"demonic" should be capitalized

Reply:
or you could just use, like, "Abyssal" instead as a D&D reference

Reply:
let curators understand the wording on the packaging, since they're supposed to know all languages

Reply:
yeah well they dont know japanese either